renovate.json

```diff
@@ -5,5 +5,6 @@
       "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
       "automerge": true
     }
-  ]
+  ],
+  "schedule": ["before 9am on monday"]
```

Comment:
what does this do?

Reply:
only opens PRs 9am on mondays, we have similar config in seaport-js and opensea-js. the natural language is a bit weird.